_data/simple-icons.json

```diff
@@ -17993,6 +17993,12 @@
             "hex": "009CFB",
             "source": "https://www.zingat.com"
         },
+        {
+            "title": "Zip Pay",
+            "hex": "1A0826",
+            "source": "https://zip.co",
+            "guidelines": "https://zip.co/investors/global-media-kit"
```

Comment:
The link is not available anymore.

Reply:
Yup the took the guidelines completely off their site, I can't find the exact same thing. But found something similar, so I'll replace that link with that I found: https://zip.co/nz/marketing/the-logo/